src/status_im/contexts/wallet/send/save_address/view.cljs

```diff
@@ -0,0 +1,104 @@
+                                 :format   :long}])
+       :on-press        (fn []
+                          (rf/dispatch [:show-bottom-sheet
+                                        {:content
+                                         (fn []
+                                           [network-preferences/view
+                                            {:on-save (fn [])
+                                             ;; (fn [chain-ids]
+                                             ;;   (rf/dispatch [:hide-bottom-sheet])
+                                             ;;   (save-account
+                                             ;;    {:account     account
+                                             ;;     :updated-key network-preferences-key
+                                             ;;     :new-value   chain-ids}))
+                                             ;; :watch-only? watch-only?
+                                            }])}]))
+       :container-style style/data-item}]]))
+
+(defn view
+  []
+  (let [[address-label set-address-label] (rn/use-state "")
+        [address-color set-address-color] (rn/use-state :blue)
+        placeholder                       (i18n/label :t/address-name)]
+    [rn/view {:style style/container}
+     [floating-button-page/view
+      {:footer-container-padding 0
+       :header                   [quo/page-nav
+                                  {:type                :no-title
+                                   :background          :blur
+                                   :icon-name           :i/close
+                                   :on-press            #(rf/dispatch [:navigate-back])
+                                   :accessibility-label :save-address-top-bar}]
+       :footer                   [quo/button
+                                  {:accessibility-label :save-address-button
+                                   :type                :primary
+                                   :container-style     style/save-address-button}
+                                  (i18n/label :t/save-address)]
+
+       :customization-color      address-color
+       :gradient-cover?          true}
+
+      [rn/view
+       {:style style/account-avatar-container}
+       [quo/user-avatar
+        {:full-name           (if (string/blank? address-label)
+                                placeholder
+                                address-label)
+         :customization-color address-color
+         :size                :big}]]
+
+      [quo/title-input
+       {:blur?               true
+        :auto-focus          true
+        :max-length          24
+        :size                :heading-1
+        :placeholder         placeholder
+        :default-value       address-label
+        :on-change-text      set-address-label
+        :customization-color address-color
+        :container-style     style/title-input-container}]
+
+      [quo/divider-line {:container-style style/divider-1}]
```

Comment:
nit perhaps title-divider is a more descriptive name here?
i.e style/title-divider}